target/dovecot/10-ssl.conf

```diff
@@ -43,9 +43,8 @@ ssl_key = </etc/dovecot/ssl/dovecot.key
 #ssl_cert_username_field = commonName
 
 # SSL DH parameters
-# Generate new params with `openssl dhparam -out /etc/dovecot/dh.pem 4096`
-# Or migrate from old ssl-parameters.dat file with the command dovecot
-# gives on startup when ssl_dh is unset.
+# This parameter will be optional in dovecot 2.3 https://wiki2.dovecot.org/Upgrading/2.3#dhparams
+# Until non-ECC based DH algorithms are dropped, please use recommended pre-defined DHE Groups https://github.com/internetstandards/dhe_groups
```

Comment:
I assume this is sourced from dovecot config? As the image is using a newer dovecot, this comment otherwise seems a bit odd for anyone that'd be seeing it for the first time? Perhaps replacing it with the [warning from dovecot docs](https://doc.dovecot.org/admin_manual/ssl/dovecot_configuration/)?

> Since v2.3.3+ Diffie-Hellman parameters have been made optional, and you are encouraged to disable non-ECC DH algorithms completely.

The 2nd line could be something like:

> `docker-mailserver` is configured to use the recommended pre-defined 4096-bit DHE Group at https://github.com/internetstandards/dhe_groups

As that is what the PR is doing rather than pulling in upstream config?